crates/wasmtime/src/runtime/component/concurrent/futures_and_streams.rs

```diff
@@ -507,13 +507,18 @@ impl<T> FutureWriter<T> {
     /// value; otherwise it will return `false`, meaning the read end was dropped
     /// before the value could be delivered.
     ///
-    /// Note that the returned `Future` must be polled from the event loop of
-    /// the component instance from which this `FutureWriter` originated.  See
-    /// [`Instance::run`] for details.
-    pub fn write(mut self, value: T) -> impl Future<Output = bool> + Send + 'static
+    /// # Panics
+    ///
+    /// Panics if the store that the `accessor` is derived from does not own
```

Comment:
```suggestion
    /// Panics if the store that the [`Accessor`] is derived from does not own
```
... just to make it consistent with the corresponding doc comments below